nyc.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  reporter: ["text", "lcov"],
+  include: ["out/src/**"],
+  exclude: ["out/src/logger/**"],
+  // https://reflectoring.io/100-percent-test-coverage/
```

Comment:
If you don't follow what's described in the link, don't link to it.
Other projects that have this link in the nyc config file actually do have 100% code coverage* as described there.

Reply:
Done